build/config/config1.json

```diff
@@ -26,5 +26,11 @@
     "enabled": true,
     "address": "0.0.0.0:9000",
     "endpoint": "/metrics"
+  },
+  "rpc": {
+    "enabled": true,
+    "port": "26657",
```

Comment:
Let's not use the tendermint port (personal ask) XD

Reply:
+1 I suggest `508832`

I converted  `pocket` to `socket`.

I then replace `s` with 5, `o` with 0 and the `e` with 3.

I decided to use `2` for t because it stars with the same letter.

I didn't what to do with `ck` so I went with `88` because Pocket is unstoppable, so we'll go to infinity and beyond.

Reply:
Love it but your enthusiasm is so elevated that overflowed the hard limit on ports (65535) :) 

Did you mean `50832`?

Reply:
lmfao. `50832` it is